### ISSUE TITLE
Guidance

### ISSUE THREAD

**Issue by foffer** [soumis/aws-sdk-osx#2]
Hi, thanks for providing this framework for os x!

Could you provide some guidance as to how to use it? I can't seem to get it to work. Do I fetch via cocoa pods?

**Comment by klmitchell2** [comment]
I am having the same issue. In the Readme it says to include the 'AWSiOSSDKv2'
 'AWSCognitoSync' pods but these or iOS compatible pods.

**Comment by jonesthai** [comment]
I agree with foffer, is there any documentation to help show how to install and use the SDK?
